### ISSUE TITLE
GU0018 don't warn when more than one mock of same type.

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#280]
```cs
var buttonMock1 = new Mock<IButtonPlc>(MockBehavior.Strict);
var buttonMock2 = new Mock<IButtonPlc>(MockBehavior.Strict);
```
